src/WebJobs.Script.WebHost/Filters/RequiresRunningHostAttribute.cs

```diff
@@ -49,10 +50,12 @@ public IFilterMetadata CreateInstance(IServiceProvider serviceProvider)
         private class RunningHostCheckAttribute : ActionFilterAttribute
         {
             private readonly IScriptHostManager _hostManager;
+            private readonly IOptionsMonitor<ScriptApplicationHostOptions> _applicationHostOptions;
```

Comment:
For non-admin apis we were already covered by HostAvailabilityCheckMiddleware which does the offline check. However the function status endpoint is an admin API so that middleware doesn't run.